.github/workflows/tox-test.yml

```diff
@@ -108,14 +108,15 @@ jobs:
           restore-keys: ${{ runner.os }}-build-owasp-
       - name: OWASP check
         run: |
-          VERSION=$(curl -s https://jeremylong.github.io/DependencyCheck/current.txt)
-          curl -L "https://github.com/jeremylong/DependencyCheck/releases/download/v$VERSION/dependency-check-$VERSION-release.zip" --output dependency-check.zip
+          VERSION=$(curl -s https://dependency-check.github.io/DependencyCheck/current.txt)
+          curl -Ls "https://github.com/dependency-check/DependencyCheck/releases/download/v$VERSION/dependency-check-$VERSION-release.zip" --output dependency-check.zip
```

Comment:
Actually one nitpick. Can you add also parameter --fail to the curl. ( -f, --fail or some variant like --fail-with-body).
Otherwise one can easily overlook failure in logs with `-s`